File: kafka-client-ex/src/main/java/in/srini91/learn/kafkaclientex/dao/repository/PersonRepository.java
package in.srini91.learn.kafkaclientex.dao.repository;

import in.srini91.learn.kafkaclientex.dao.model.Person;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface PersonRepository extends JpaRepository<Person,Long> {
}

File: kafka-cloud-stream-ex/src/main/kotlin/srini/learn/kafka/kafkacloudstreamex/dao/repository/PersonRepository.kt
package srini.learn.kafka.kafkacloudstreamex.dao.repository

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import srini.learn.kafka.kafkacloudstreamex.dao.model.Person
@Repository
interface PersonRepository:JpaRepository<Person,Long> {
}